Lesson5/Dariia-Skoryk/task6_set/task/main.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <set>
+#include <string>
+#include <vector>
+
+void toLower(std::string &stringToLower) {
+    for (char & entry: stringToLower) {
+        entry = entry <= 'Z' && entry >= 'A'? entry - 'A' + 'a' : entry;
+    }
+}
+
+std::vector<std::string> getUniqueWords(const std::string & text) {
+    size_t indexOfBegin, indexOfEnd, currentPosition{0};
+    std::string word, punctuation(" ");
```

Comment:
never combine variables in one line. separate it:
std::string word;
std::string punctuation.

Also, please initialize variables near the place of their usage, not at the begin of the method.